STYLE-GUIDE.md

```diff
@@ -51,8 +51,8 @@ the body entirely); tools like rebase can get confused if you run the
 two together.
 
 Write your commit message in the imperative: "Fix bug" and not "Fixed
-bug."  This convention matches up with commit messages generated by
-commands like git merge and git revert.
+bug". This convention matches up with commit messages generated by
```

Comment:
Periods and commas are always placed inside the quotations marks. (http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html)  However, do not apply this to the next line, as those aren't considered quotes.